docs/cli-help.txt

```diff
@@ -551,6 +551,17 @@ dns-dnsmadeeasy:
   --dns-dnsmadeeasy-credentials DNS_DNSMADEEASY_CREDENTIALS
                         DNS Made Easy credentials INI file. (default: None)
 
+dns-exoscale:
```

Comment:
I don't think this file is typically updated as a part of this sort of commit; I think it's updated as a part of the release process. (@bmw?)

Reply:
> I think it's updated as a part of the release process.

That's correct. We do it this way to help make sure that documentation for unreleased features don't make their way to our site.

That fact that we do this is not very clear though.

Reply:
fair enough